libmscore/edit.cpp

```diff
@@ -2148,7 +2148,7 @@ void Score::deleteMeasures(MeasureBase* is, MeasureBase* ie, bool preserveTies)
       if (ie->isMeasure()) {
             Measure* iem = toMeasure(ie);
             if (iem->isMMRest())
-                  ie = iem = iem->mmRestLast();
+                  ie = /*iem = */iem->mmRestLast();
```

Comment:
Why uncommenting rather than deleting thise?

Reply:
Because I thought it might me necessary in the lines below it that are commented out too.